src/test/java/xyz/vegaone/easytracking/task/TaskServiceTest.java

```diff
@@ -38,9 +38,7 @@
     @Autowired
     private UserStoryService userStoryService;
 
-    @Test
-    public void createTaskTest() {
-        //given
+    private UserStory buildAndSaveUserStory(){
```

Comment:
The same comment as in the `BugServiceTest`. Would be best to save the user story and return it.